HIP/hip-367.md

```diff
@@ -0,0 +1,238 @@
+---
+hip:367
+title: HIP Unlimited token associations per Account
+author: Anirudh Ghanta <anirudh.ghanta@hedera.com>
+working-group: Richard Bair<richard.bair@hedera.com>, Jasper Potts<jasper.potts@hedera.com>, Michael Tinker<michael.tinker@hedera.com>, Anirudh Ghanta<anirudh.ghanta@hedera.com>
+type: Standards Track
+category: Service
+needs-council-approval: Yes
+status: Draft
+created: 2022-02-17
+discussions-to: [Discussion thread](https://github.com/hashgraph/hedera-improvement-proposal/discussions/380)
+updated: 2202-03-11
+---
+
+## Abstract
+Permits every account to hold an unlimited number of token associations. Retains the current pricing for each token association. Modifies the number of token associations returned in the `getAccountInfo` and `getAccountBalance` query. Changes the criteria for `cryptoDelete` transaction to successfully mark the account as `deleted`
+
+## Motivation
+The current Services API permits the user to have at most 1,000 token associations per account.
+A user can work around this limitation by creating multiple accounts, each of which is limited to
+1,000 token associations. For some use cases, this 1,000 token association limit is frustrating.
+Due to the current implementation, it makes state proofs large and expensive. In addition, the
```

Comment:
I'm not sure any state proofs will get smaller after this HIP; am I missing something? 🤔